dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -199,11 +211,11 @@ public void testForLogDays() throws Throwable {
 
     ZonedDateTime expectedStart =
         nowAtUTC
-            .truncatedTo(ChronoUnit.HOURS)
+            .truncatedTo(ChronoUnit.SECONDS)
```

Comment:
I would suggest to avoid making tests dependent on the current time (`now`).

Reply:
The default `Iterable` depends on the current time atm.

Reply:
The `ZonedIntervalIterable` class does not depend on the current time. I would suggest to avoid referencing the `ZonedIntervalIterableGenerator` class in the `ZonedIntervalIterableTest` test class.

I would suggest to mock the time in the tests of `ZonedIntervalIterableGenerator` class.

Reply:
Added it to the TODO list